app/assets/javascripts/views/data_portal/slides/ShareView.js

```diff
@@ -0,0 +1,67 @@
+(function (App) {
+  App.View.ShareView = App.Component.Slide.extend({
+
+    template: JST['templates/data_portal/slides/shareWidget'],
+
+    className: 'slide',
+
+    events: {
+      'click .js-back': '_onBack',
+      'click .js-copy': '_onCopy',
+      'blur  .js-copy': '_onBlur'
+    },
+
+    initialize: function (settings) {
+      this.constructor.__super__.initialize.call(this, settings);
+    },
+
+    _onBack: function () {
```

Comment:
This is useless too. Instead of calling `_onBack`, call directly `this.onCallback`.